yaml_files/simphony_metadata.yml

```diff
@@ -503,32 +496,21 @@ CUDS_KEYS:
 
   INLET_OUTLET:
     definition: Inlet outlet boundary condition (outlet condition is zero gradient and inlet given variable value)
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.MATERIAL:
-      shape: (:)
-      default: []
 
   WETTING_ANGLE:
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     definition: Volume fraction wall boundary condition with specified contact angle
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.CONTACT_ANGLE:
-      default: 45.0
+    variable: CUBA.VOLUME_FRACTION
```

Comment:
by specifications: anything in lowercase is fixed in the metadata level, so variable is equivalent to CUBA.VARIABLE. 
we should probably at some point formalise the plural (s) use! If now variables is equivalent to. 
CUBA.VARIABLE: 
  shape: (:) 
then we can use variables.